appserver/payara-appserver-modules/hazelcast-eclipselink-coordination/src/main/java/fish/payara/persistence/eclipselink/cache/coordination/HazelcastTopicStorage.java

```diff
@@ -0,0 +1,270 @@
+/*
```

Comment:
Rather than using Hazelcast directly here is it possible to use the wrapper service https://github.com/payara/Payara/blob/master/nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/eventbus/EventBus.java

Reply:
Sure.

Reply:
@smillidge You have attached this change request to a line in the header. This won't close automatically.